design-system/react/src/components/Button/Button.tsx

```diff
@@ -79,6 +79,25 @@ function getChildren({
   );
 }
 
+function getPropsToOmit(props: t.ButtonDef['props']) {
+  const toOmit = [];
+  if (!props.onKeyDown) toOmit.push('onKeyDown');
+  if (!props.onKeyUp) toOmit.push('onKeyUp');
+  if (!props.onKeyPress) toOmit.push('onKeyPress');
+  if (!props.onFocus) toOmit.push('onFocus');
+  if (!props.onBlur) toOmit.push('onBlur');
+  if (!props.onMouseDown) toOmit.push('onMouseDown');
+  if (!props.onDragStart) toOmit.push('onDragStart');
+  if (!props.onPointerDown) toOmit.push('onPointerDown');
+  if (!props.onPointerUp) toOmit.push('onPointerUp');
+  if (!props.onPress) toOmit.push('onPress');
+  if (!props.onPressStart) toOmit.push('onPressStart');
+  if (!props.onPressEnd) toOmit.push('onPressEnd');
+  if (!props.onPressChange) toOmit.push('onPressChange');
+  if (!props.onPressUp) toOmit.push('onPressUp');
+  return toOmit;
+}
```

Comment:
This is temporary fix because of the usage of the `useButton` is getting accessibility issue, in order to don't lost so much times here I just omit this auto-generated `useButton` handlers by now